tests/queries/0_stateless/03280_aliases_override_logic.sql

```diff
@@ -0,0 +1,27 @@
+SELECT b FROM
+(
+    SELECT number, number*2
+    FROM numbers(2)
+) as x (a, b);
+
+SELECT a FROM
+(
+    SELECT number, number*2
+    FROM numbers(2)
+) as x (a, b);
+
+SELECT a FROM
+(
+SELECT number, number*2
+FROM numbers(10)
+) as x (a); -- { serverError BAD_ARGUMENTS }
+
+CREATE VIEW IF NOT EXISTS test_view_03280 (a,b) AS SELECT 1, 2;
```

Comment:
So if `test_view_03280` somehow exists already and it is defined differently, we'll run into trouble.

Better do `DROP VIEW IF EXISTS` followed by `CREATE VIEW` (without `IF EXISTS`)